### PR TITLE
Remove Deprecated API and Mpl Core from Token Metadata

### PR DESCRIPTION
**Pull request by lorisleiva**

### CODE REVIEW

File: token-metadata/js/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+```shell
+npm install @metaplex-foundation/mpl-token-metadata --save
+```
+
+This will use the new auto-generated API. If you want to use the old deprecated API, you can use the following:
+
+```shell
+npm install @metaplex-foundation/mpl-token-metadata@^1.2 --save
+```
```
Comment (thlorenz) [inline token-metadata/js/README.md:13]: Should add something along the lines of the below:

~~~

In order to start using the new API but still have access to the deprecated one please do the following:

```sh
npm install @metaplex-foundation/mpl-token-metadata@~2.0 --save
```

```sh
import { deprecated } from @metaplex-foundation/mpl-token-metadata
```

**Review by thlorenz — CHANGES_REQUESTED** [review]
Just add the extra info to the readme and it's ready to merge once CI passes

File: token-metadata/js/README.md (hunk 1 above)
Comment (lorisleiva) [reply token-metadata/js/README.md:13]: Do we really want to do that though if `2.0` is not really working with the new API due to `spl-token` being forced to be the old version (0.1)?

Comment (thlorenz) [reply token-metadata/js/README.md:13]: It's just to explain an option for people ... they could use different spl-tokens in the same app .. it's just an issue once you bundle in the browser potentially afaik.

But I'm fine either way .. it's mainly also so we can link people to it if we get asked.

Comment (lorisleiva) [reply token-metadata/js/README.md:13]: Got it, I'll add now :)

**Review by thlorenz — APPROVED** [review]
LGTM I'll merge and publish once CI passes

**Review by thlorenz — APPROVED** [review]

**Comment by oboxodo** [comment]
@lorisleiva The Solana Cookbook is still recommending to use the deprecated (and deleted in this PR) `Metadata.findDataByOwner` [here](https://solanacookbook.com/references/nfts.html#how-to-get-all-nfts-from-a-wallet).

Any suggested alternative to do that besides copying all the deleted code?

**Comment by lorisleiva** [comment]
Hi Diego 👋  

We are working on a brand new JS SDK for Metaplex. You can already use it to get all the NFTs of a given owner. 

Here's the documentation: https://github.com/metaplex-foundation/js-next#findnftsbyowner

I hope this helps. 🍀

**Comment by oboxodo** [comment]
Thanks for your prompt reply, @lorisleiva. We'll check that out soon.

**Comment by acrens** [comment]
> Hi Diego 👋
> 
> We are working on a brand new JS SDK for Metaplex. You can already use it to get all the NFTs of a given owner.
> 
> Here's the documentation: https://github.com/metaplex-foundation/js-next#findnftsbyowner
> 
> I hope this helps. 🍀

@lorisleiva Great, when is it planned to be available for production?

**Pull request merged**